src/shaderlib/picking/picking.js

```diff
@@ -0,0 +1,95 @@
+const DEFAULT_MODULE_OPTIONS = {
+  pickingSelectedColor: null, //  Set to a picking color to visually highlight that item
+  pickingThreshold: 1.0,
+  pickingActive: false // Set to true when rendering to off-screen "picking" buffer
+};
```

Comment:
yeah, luma examples would have to set this at least once before first draw. Normally highlight color is fixed and doesn't get updated as frequently as other uniforms.